apps/pythonapi/examples/dataset_example.py

```diff
@@ -0,0 +1,54 @@
+# %% [md]
+#
+# # Opening Datasets
+#
+# Vapor supports a variety of scientific data formats.
```

Comment:
This comment is a little misleading. The example doesn't show how to open one of VAPOR's supported data formats. Instead it shows how to load a synthetic data set. How about co-opting Pythia's DATASETS.fetch() method, as used [here](https://foundations.projectpythia.org/core/xarray/xarray-intro.html#access-netcdf-data-with-xr-open-dataset), so that we can easily include simulation data sets?

Reply:
I don't quite follow, there is a list of examples on lines 21-23?

Reply:
Yes, but these commented out examples aren't runnable, and we don't provide any direction to the user on how to retrieve the listed data sets which are known to work in this script. Since the example is intended to demonstrate "Opening Datasets" a more helpful script (notebook) would be executable with no changes required, and would open an actual data set that is stored in a file.

If you look at examples in GeoCAt, Pythia, Xarray, MetPy, etc. you will see that most of them make use of some form of pooch to automatically retrieve sample data from a GitHub repo, which, among other things allows the script to be run without worrying about file paths. 

I'd be fine with marking this as resolved and opening a separate issue to address this.

Reply:
My primary concern is that, since Vapor uses such old versions of many of its third party libraries, adding additional modern dependencies like Pythia is finicky since it likely depends on newer builds. Because of this difficulty with Conda, If we want to do this I would ask that we postpone it and open a separate issue like you suggested.

To address your points: The current notebook is executable with no changes required. The datasets in those lines are not real so they cannot be retrieved. If I replaced them with real datasets and made those lines runnable what would you like me to do with all the open datasets?

Reply:
@stasj I'm fine with postponing the changes I'm suggesting. Please open an issue. 

However, I'm more concerned now with the difficulties you've mentioned regarding our library dependencies. I don't fully understand the issue, but it is going to be important for the vapor python package to play well with other packages in the scientific python ecosystem. We just went though the process of updating some, but not all, of our 3rd party libs. Do we need to bite the bullet and update them all?

cc: @sgpearse

Reply:
Currently our Conda third party dependencies are locked to the same versions (or at least very close) to our manually built libraries. We may need to relax these requirements to improve compatibility with other packages.

Reply:
I'm going to mark this as resolved and open a separate issue with regard to compatibility with other packages.